### ISSUE TITLE
welcome video breaks login grafics

### ISSUE THREAD

**Issue by dogi** [open-learning-exchange/BeLL-Apps#99]

**Comment by SadiaRasheed** [comment]
I need to change the attributes of the page after the "Welcome" button appears.

**Comment by MWakas** [comment]
Sadia, how much effort required for this change, hours or day?

**Comment by SadiaRasheed** [comment]
login graphics are working fine now. Please see the screenshots given below
![99-1](https://cloud.githubusercontent.com/assets/10362799/10190536/c1511184-6786-11e5-8925-3c9b319cb691.PNG)
![99-2](https://cloud.githubusercontent.com/assets/10362799/10190538/c59be692-6786-11e5-8b4a-8cf274bfad37.PNG)

**Comment by jstearnsw** [comment]
Hi Sadia,

A few comments on the different layouts:

**New community without welcome video**:
Let's do away with this layout option - we prefer a combination of options from the other communities (outlined below)
![new comm without welcome](https://cloud.githubusercontent.com/assets/11721132/10256133/13118174-691d-11e5-80bb-b53dd5da9c44.JPG)

**Old community with welcome video**:
We like the green color for the "Welcome" button - we would like to use this color green for the final layout "Welcome" and "Sign In" buttons
![old comm with welcome](https://cloud.githubusercontent.com/assets/11721132/10256134/131a3698-691d-11e5-97c8-d4d92a8507a6.JPG)

**New community with welcome video**:
We like this layout/look
![new comm with welcome](https://cloud.githubusercontent.com/assets/11721132/10256132/130a3400-691d-11e5-8a82-5de14ec90975.JPG)

**Old community without welcome video**:
We like this layout/look
![old comm without welcome](https://cloud.githubusercontent.com/assets/11721132/10256135/131a676c-691d-11e5-93f9-b31ddb895f83.JPG)

To recap, we would like to return to this option for a login screen without welcome video:
![old comm without welcome](https://cloud.githubusercontent.com/assets/11721132/10256135/131a676c-691d-11e5-93f9-b31ddb895f83.JPG)

And keep this option for the login screen with welcome video:
![new comm with welcome](https://cloud.githubusercontent.com/assets/11721132/10256132/130a3400-691d-11e5-8a82-5de14ec90975.JPG)

However, for both we would like the green buttons to be changed to the Welcome button color (outlined in red) from this login page:
![old comm with welcome](https://cloud.githubusercontent.com/assets/11721132/10257182/e757a4ee-6923-11e5-83f2-f66eff1ca4d1.JPG)

Let me know if you need any more clarification

**Comment by SadiaRasheed** [comment]
please see screenshots below
With Welcome Video
![with welcome video](https://cloud.githubusercontent.com/assets/10362799/10282406/14d56d26-6b94-11e5-9eb6-844038f2ef85.PNG)

Without Welcome Video
![without welcome video](https://cloud.githubusercontent.com/assets/10362799/10282407/19b14536-6b94-11e5-84df-f072793ce8b7.PNG)

Please let me know if that what you wanted

**Comment by SadiaRasheed** [comment]
github commit link
https://github.com/open-learning-exchange/BeLL-Apps/commit/c398c3e0a382cc057cffc861e31a6a784ea7e71c
